packages/contracts-bedrock/src/cannon/PreimageOracle.sol

```diff
@@ -195,6 +200,9 @@ contract PreimageOracle is IPreimageOracle {
         // The part offset must be within the bounds of the claimed size + 8.
         if (_partOffset >= _claimedSize + 8) revert PartOffsetOOB();
 
+        // The claimed size must be at least 1.8MB.
```

Comment:
maybe qualify that this is typically the case for mainnet/testnets or something.

Reply:
Ah whoops - Forgot to modify this comment, good catch.